### ISSUE TITLE
Question: How to use multiple orm's together?

### ISSUE THREAD

**Issue by pratik60** [DatabaseCleaner/database_cleaner#424]
I can't figure out how to use multiple orm's together. We initially used only ActiveRecord, and database_cleaner worked great.

Recently we introduced mongodb to the stack, but am unable to figure out the configuration to have mongoid and active record using database cleaner.

Our current database_cleaner.rb looks like this -: 

```
RSpec.configure do |config|
  config.use_transactional_fixtures = false

  config.before(:suite) do
    DatabaseCleaner.clean_with(:truncation)
  end

  config.before(:each) do
    DatabaseCleaner.strategy = :transaction
  end

  config.before(:each, js: true) do
    DatabaseCleaner.strategy = :truncation
    FactoryGirl.reload
  end

  config.before(:each) do
    DatabaseCleaner.start
  end

  config.append_after(:each) do
    DatabaseCleaner.clean
  end
end
```

**Comment by eduzera** [comment]
i'm in the same situation :/

**Comment by nbulaj** [comment]
@pratik60 did you read "_How to use with multiple ORM's_" in the docs? ( https://github.com/DatabaseCleaner/database_cleaner#how-to-use-with-multiple-orms )

Set up all your connections via `DatabaseCleaner[:orm, { :connection => :something }]`

**Comment by etagwerker** [comment]
@eduzera @pratik60 Did you try with the example mentioned in the documentation? Please let me know.

**Comment by pratik60** [comment]
Updated it to this
```
RSpec.configure do |config|
  config.use_transactional_fixtures = false

  config.before(:suite) do
    DatabaseCleaner[:active_record].strategy = :truncation
    DatabaseCleaner[:mongoid].strategy = :truncation
  end

  config.before(:each) do
    DatabaseCleaner[:active_record].strategy = :truncation
    DatabaseCleaner[:mongoid].strategy = :truncation
  end

  config.before(:each, js: true) do
    DatabaseCleaner[:active_record].strategy = :truncation
    DatabaseCleaner[:mongoid].strategy = :truncation
    FactoryGirl.reload
  end

  config.before(:each) do
    DatabaseCleaner.start
  end

  config.append_after(:each) do
    DatabaseCleaner.clean
  end
end

```

and in each of them I now get

```
12) DpaController Product Feed fetch product feeds
      Failure/Error: Unable to find matching line from backtrace
      NoMethodError:
        undefined method `default_session' for Mongoid:Module
      # /Users/pratikbothra/.rvm/gems/ruby-2.2.2/gems/database_cleaner-1.4.1/lib/database_cleaner/mongoid/truncation.rb:30:in `session'
      # /Users/pratikbothra/.rvm/gems/ruby-2.2.2/gems/database_cleaner-1.4.1/lib/database_cleaner/moped/truncation_base.rb:26:in `collections'
      # /Users/pratikbothra/.rvm/gems/ruby-2.2.2/gems/database_cleaner-1.4.1/lib/database_cleaner/moped/truncation_base.rb:14:in `clean'
      # /Users/pratikbothra/.rvm/gems/ruby-2.2.2/gems/database_cleaner-1.4.1/lib/database_cleaner/base.rb:86:in `clean'
      # /Users/pratikbothra/.rvm/gems/ruby-2.2.2/gems/database_cleaner-1.4.1/lib/database_cleaner/configuration.rb:79:in `block in clean'
      # /Users/pratikbothra/.rvm/gems/ruby-2.2.2/gems/database_cleaner-1.4.1/lib/database_cleaner/configuration.rb:79:in `each'
      # /Users/pratikbothra/.rvm/gems/ruby-2.2.2/gems/database_cleaner-1.4.1/lib/database_cleaner/configuration.rb:79:in `clean'
      # ./spec/support/database_cleaner.rb:28:in `block (2 levels) in <top (required)>'
      # /Users/pratikbothra/.rvm/gems/ruby-2.2.2/gems/rspec-retry-0.4.4/lib/rspec/retry.rb:53:in `block (3 levels) in apply'
      # /Users/pratikbothra/.rvm/gems/ruby-2.2.2/gems/rspec-retry-0.4.4/lib/rspec/retry.rb:44:in `times'
      # /Users/pratikbothra/.rvm/gems/ruby-2.2.2/gems/rspec-retry-0.4.4/lib/rspec/retry.rb:44:in `block (2 levels) in apply'
```

**Comment by pratik60** [comment]
@etagwerker I believe it is an issue with mongoid 3.2 as well

**Comment by pratik60** [comment]
Seems fixed after upgrading to 1.5.1. 

Can we do a new release as well?
